ansible/configs/sap-integration/files/kubelet-bootstrap-cred-manager-ds.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
```

Comment:
Do you even need this? You're using a version of OpenShift that supports shutdown and resume (4.4.8+) and this was for a workaround.

Reply:
Sure, not sure it was used for any specific thing in the config mine is based on